test-files/terraform/aws/ec2/ec2_instance.tf

```diff
@@ -0,0 +1,94 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["123456789123"] # Canonical
+}
+
+resource "aws_instance" "web" {
```

Comment:
In our test.yml workflow, is there a way to avoid having these checks block the PR, i.e., handle the exit code we'd expect?

Reply:
It's not the exit code that is blocking, it's that we have it writing the annotations since this marks it as `action_required`.

We could swap the test to be a manually executed workflow instead of on the PR?